packages/ibm-products/src/components/Coachmark/Coachmark.tsx

```diff
@@ -299,7 +299,15 @@ export let Coachmark = forwardRef<HTMLElement, CoachmarkProps>(
             <Popover
               highContrast
               caret
-              style={{ position: 'absolute', ...overlayPositionStyle }}
+              ref={(el) => {
```

Comment:
Can we convert this to a const using useRef just incase in the future somebody wants to use a ref with this and don't have to refactor this. Thanks!